### DISCUSSION TITLE
指定端口注入后永远都是默认的19088,跪请指点!

### DISCUSSION THREAD

**Discussion by qwe159357890** [ttttupup/wxhelper#115] [Q&A]
![Snipaste_2023-04-27_09-02-48](https://user-images.githubusercontent.com/97860760/234733791-4dd2e399-c68a-468c-828a-c197869aed28.png)

**Comment by qwe159357890** [comment]
![aa](https://user-images.githubusercontent.com/97860760/234733909-fef9801f-baa2-401d-a12d-1c6999f992bf.png)
用的是最新发布的这个版本的工具

**Comment by ttttupup** [comment]
指定端口的需要指定版本的dll，用最新的可以配置文件配置端口。

**Comment by qwe159357890** [comment]
大佬您真实神速!!!  弱弱问一句 我看好多人问关于注入问题了(所有Issues我都翻看过)  是不是能考虑下简化下,毕竟我们这种不懂c++的小白好多! 说实话其他都很好很好,就是注入弄得有点晕,太多版本了. 我看您回复好多人(指定端口的需要指定版本的dll) 这句话  到现在我也没弄明白 这句话什么意思

**Comment by qwe159357890** [comment]
比如像您提到的 [ComWeChatRobot](https://github.com/ljc545w/ComWeChatRobot) 大佬那样,运行前先启动个exe 然后注入功能放到client.py客户端里面!

**Comment by ttttupup** [comment]
这种dll本来就不是给小白用的。个人精力有限。
